### DISCUSSION TITLE
[.NET | Infra] Recognizers not working if certain dependencies are updated manually

### DISCUSSION THREAD

**Discussion by 7i77an** [microsoft/Recognizers-Text#2238] [General]
When update dependencies, recognizers not return any results:

List of dependencies to update:

Microsoft.Extensions.Caching.Abstractions v3.1.6 (installed 2.2.0)
Microsoft.Extensions.Caching.Memory v3.1.6 (installed 2.2.0)
Microsoft.Extensions.DependencyInjection.Abstractions v3.16 (installed 2.2.0)
Microsoft.Extensions.Options v3.16 (installed 2.2.0)
Microsoft.Extensions.Primitives v3.16 (installed 2.2.0)
System.ComponentModel.Annotations v4.70 (installed 4.5)
System.Runtime.CompilerServices.Unsafe v4.71 (installed 4.5.3)

If don't update dependencies, recognizers work ok.

 - Platform: [.NET]
 - Environment: [nuget package]
 - Version of package [v1.4.0]
- IDE: [Visual Studio 2017]

**Comment by tellarin** [comment]
Do you see any errors? Unfortunately this update is not currently planned in the main repo as it introduces downstream breaking changes. So we won't be able to look into it.
